src/cursor/cur_layered.c

```diff
@@ -245,7 +245,7 @@ __clayered_open_cursors(WT_CURSOR_LAYERED *clayered, bool update)
     if (clayered->stable_cursor == NULL) {
         if (!S2C(session)->layered_table_manager.leader) {
             ckpt_cfg[0] = WT_CONFIG_BASE(session, WT_SESSION_open_cursor);
-            ckpt_cfg[1] = "checkpoint=" WT_CHECKPOINT ",raw,checkpoint_use_history=false";
+            ckpt_cfg[1] = ",raw,checkpoint_use_history=false,force=true";
             ckpt_cfg[2] = NULL;
```

Comment:
Why we can't remove the other configs as well and just do

ret = __wt_open_cursor(
              session, layered->stable_uri, &clayered->iface, NULL, &clayered->stable_cursor);

Reply:
We need the `force=true`, so it can't just be `NULL`. The rest of the config is optional as far as I can tell, but it was already there so there wasn't a reason to touch it.